pkgs/development/libraries/libunwind/default.nix

```diff
@@ -19,7 +19,9 @@ stdenv.mkDerivation rec {
     })
   ];
 
-  postPatch = lib.optionalString stdenv.hostPlatform.isMusl ''
+  postPatch = if stdenv.cc.isClang then ''
```

Comment:
@sternenseemann I guess it does not make sense to compile libunwind for clang at all than?

Reply:
It might, I really don't know.

Reply:
Seems though: https://clang.llvm.org/docs/Toolchain.html#libunwind-nongnu-org